Units/deploy-tools/deploy-tools.sh

```diff
@@ -416,7 +415,7 @@ function deploy-tools.parseArguments() {
   # Print help
   if [ "$HELP" = true ]; then
     deploy-tools.printUsage
-    exit 1
+    exit 0
```

Comment:
Ich habe neuerdings gelernt, dass das Anzeigen der Usage als Fehler gewertet wird, da man unter Umständen den Befehl falsch angegeben hat. Wollen wir das auch so handhaben, oder ist das in deinen Augen nicht sinnvoll? Jedenfalls habe ich deshalb die exit Codes so angepasst, wusste jedoch nicht, dass Jenkins dadurch auf die Nase fällt 😅

Reply:
Ich denke, wenn die Usage gewollt mit "-h" angezeigt wird, ist das kein Fehler, oder?
Wenn ein Fehler vorliegt und daraufhin auch die Usage angezeigt wird, dann ist das ein Fehler.